src/main/java/com/redhat/cloud/notifications/db/ApplicationResources.java

```diff
@@ -71,6 +71,15 @@
                 .getSingleResultOrNull();
     }
 
+    public Uni<EventType> getEventType(String bundleName, String applicationName, String eventTypeName) {
```

Comment:
@gwenneg could you review this method?  I rebased the code and I'm using the hibernate API.

Reply:
This method LGTM 👍